packages/app/lib/ui/screens/create_sighting.dart

```diff
@@ -21,188 +18,97 @@ class CreateNewScreen extends StatefulWidget {
-      return retrieveError;
-    }
-    if (_image != null) {
-      return Container(
-        height: 300,
-        child: Image.file(
-          _image!,
-          fit: BoxFit.cover,
+    return MeliScaffold(
+      title: 'Create new',
+      floatingActionButtons: [
+        // MeliFloatingActionButton(
+        //     heroTag: 'create_new',
+        //     icon: Icon(Icons.camera_alt_outlined),
+        //     onPressed: () {
+        //       MeliCameraProviderInherited.of(context).capturePhoto();
+        //     }),
+        ExpandableFab(
+          icon: const Icon(Icons.add_a_photo),
+          expandDirection: ExpandDirection.right,
+          distance: 80,
+          children: [
+            ActionButton(
+              onPressed: cameraImageProvider.pickFromGallery,
+              icon: const Icon(Icons.insert_photo_outlined),
+            ),
+            ActionButton(
+              onPressed: cameraImageProvider.capturePhoto,
+              icon: const Icon(Icons.camera_alt_outlined),
+            ),
+          ],
         ),
-      );
-    } else {
-      return const Text(
-        'You have not yet picked an image.',
-        textAlign: TextAlign.center,
-      );
-    }
-  }
-
-  Widget _previewImage() {
-    return defaultTargetPlatform == TargetPlatform.android
-        ? FutureBuilder<void>(
-            future: _retrieveLostData(),
-            builder: (BuildContext context, AsyncSnapshot<void> snapshot) {
-              switch (snapshot.connectionState) {
-                case ConnectionState.none:
-                case ConnectionState.waiting:
-                  return const Text(
-                    'You have not yet picked an image.',
-                    textAlign: TextAlign.center,
-                  );
-                case ConnectionState.done:
-                  return _handlePreview();
-                case ConnectionState.active:
-                  if (snapshot.hasError) {
-                    return Text(
-                      'Pick image/video error: ${snapshot.error}}',
-                      textAlign: TextAlign.center,
-                    );
-                  } else {
-                    return const Text(
-                      'You have not yet picked an image.',
-                      textAlign: TextAlign.center,
-                    );
-                  }
+        MeliFloatingActionButton(
+            heroTag: 'submit',
+            icon: Icon(Icons.check),
+            onPressed: () async {
+              if (_formKey.currentState!.validate()) {
+                // Create sighting data
+                try {
+                  // TODO: populate all fields from form
+                  DateTime datetime = DateTime.now();
+                  await createSighting(datetime, 0.0, 0.0, [], null, null,
+                      "Some comment about this sighting");
+
+                  // Remove captured images from the provider
+                  cameraImageProvider.removeAll();
```

Comment:
If we wanna do this in a `dispose` method I think the place for it would be in the `MeliCameraProviderState` widget. As I understand it you normally want to access inherited widgets in the `build` method.

Only problem there is that as the whole app is wrapped in the image provider then dispose won't get called when navigating away from the create sightings screen. 

Think this would require moving the wrapping down a little further in the tree.

Reply:
Why not in the dispose method of the `CreateSighting` widget?

Reply:
[device-2023-10-02-173002.webm](https://github.com/p2panda/meli/assets/1822473/2c10a4a1-ddcf-47fb-9c4a-f4a74722f74b)

I've made some experiments, It is possible to enter the view and see old pictures, either when cancelling the camera view or pressing the back button in the Android bar. I think if we somehow can clear the images in the provider whenever the create sighting view is exited, that would be ideal.

Reply:
Think the behaviour we want is: You can only enter the quick sightings view when you've selected _at least_ one image. When you cancel or press back the user probably just wants to stop going through the flow.

Reply:
Nice, made this change.

With this there is no way to get to the create sighting form without taking a picture (it was janky before, but there was a way), not sure if that's the behaviour we want. Let's see as we use the app a bit, might be that we have to have an expandable fab on the homepage too.